stable/fairwinds-insights/values.yaml

```diff
@@ -405,6 +405,23 @@ truncateWorkloadMetrics:
     # -- The user ID to run the truncating workload metrics job under.
     runAsUser: 10324
 
+actionItemsFiltersRefresherCronJob:
+  # -- Resources for the action-items filters refresher job.
+  resources:
+    limits:
+      cpu: 250m
+      memory: 512Mi
+    requests:
+      cpu: 40m
+      memory: 32Mi
```

Comment:
probably want more than this. I'd set it to the same thing you have for `limits` (which is usually best practice for prod)

Reply:
done